assignment5/static/app.js

```diff
@@ -0,0 +1,47 @@
+/*global window, document*/
+(function (){
+  const section = {
+    init: function (){
+      const route = window.location.hash;
+      if(route === '') {
+        routes.pages.forEach(function (page) {
+          page === 'home'
+          ? document.getElementById(page).classList.remove('invisible')
+          : document.getElementById(page).classList.add('invisible');
+        });
+        return false;
+      } else {
+        routes.pages.forEach(function (page) {
+          `#${page}` === route
+          ? document.getElementById(page).classList.remove('invisible')
+          : document.getElementById(page).classList.add('invisible');
+        });
+      }
+    },
+    toggle: function (){
+      const route = window.location.hash;
+      routes.pages.forEach(function (page) {
+        `#${page}` === route
+        ? document.getElementById(page).classList.remove('invisible')
+        : document.getElementById(page).classList.add('invisible');
+      });
+    }
+  };
+  const routes = {
+    pages: [
+      'home',
```

Comment:
You could've put the hashes in here already. Then you wouldn't have to this to add the hash later.
```
`#${page}` === route
```